lib/snapshot/BUILD.gn

```diff
@@ -2,16 +2,15 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("$flutter_root/lib/ui/dart_ui.gni")
 import("//build/compiled_action.gni")
 import("//third_party/dart/utils/compile_platform.gni")
+import("$flutter_root/lib/ui/dart_ui.gni")
 
 if (is_fuchsia) {
```

Comment:
I'm wondering if any of these imports are actually still needed? Could you possibly try to remove them?